tests/components/sense/test_sensor.py

```diff
@@ -63,6 +70,44 @@ async def test_device_power_sensors(
     assert state.state == f"{DEVICE_2_POWER:.1f}"
 
 
+async def test_device_energy_sensors(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    mock_sense: MagicMock,
+    config_entry: MockConfigEntry,
+) -> None:
+    """Test the Sense device power sensors."""
+    await setup_platform(hass, config_entry, SENSOR_DOMAIN)
+    device_1, device_2 = mock_sense.devices
+
+    state = hass.states.get(f"sensor.{DEVICE_1_NAME.lower()}_daily_energy")
+    assert state.state == f"{DEVICE_1_DAY_ENERGY:.0f}"
+
+    state = hass.states.get(f"sensor.{DEVICE_2_NAME.lower()}_daily_energy")
+    assert state.state == f"{DEVICE_2_DAY_ENERGY:.0f}"
+
+    device_1.energy_kwh[Scale.DAY] = 0
+    device_2.energy_kwh[Scale.DAY] = 0
+    async_fire_time_changed(hass, utcnow() + timedelta(seconds=600))
```

Comment:
please use the freezer
```suggestion
    freezer.tick(timedelta(minutes=10))
    async_fire_time_changed(hass)
```